app/_layouts/plugin.html

```diff
@@ -107,6 +121,9 @@
     {% if page.params.consumer_id %}
       <tr><td><code>consumer_id</code></td><td></td><td>The id of the Consumer which this plugin will target.</td></tr>
     {% endif %}
+    {% if page.params.api_id %}
+      <tr><td><code>api_id</code></td><td></td><td>The id of the API which this plugin will target. <strong>Not available since Kong 0.14.0.</strong></td></tr>
```

Comment:
APIs will not be dropped in Kong 0.14.0, only the documentation on the Admin API